Fix Lint Warnings

This is the result of running `pod spec lint WordPressKit.podspec`:

>  -> WordPressKit (1.5.0)
>     - WARN  | url: There was a problem validating the URL http://apps.wordpress.com.
>     - NOTE  | xcodebuild:  note: Using new build system
>     - NOTE  | [iOS] xcodebuild:  note: Planning build
>     - NOTE  | [iOS] xcodebuild:  note: Constructing build description
>     - NOTE  | [iOS] xcodebuild:  warning: Skipping code signing because the target does not have an Info.plist file. (in target 'App')
>     - WARN  | xcodebuild:  wpxmlrpc/WPXMLRPC/WPXMLRPCEncoder.h:50:9: warning: parameter 'parameters' not found in the function declaration [-Wdocumentation]
>     - NOTE  | xcodebuild:  wpxmlrpc/WPXMLRPC/WPXMLRPCEncoder.h:50:9: note: did you mean 'params'?
>     - WARN  | xcodebuild:  wpxmlrpc/WPXMLRPC/WPXMLRPCEncoder.h:92:3: warning: '@param' command used in a comment that is not attached to a function declaration [-Wdocumentation]

We should clean this up to build cleanly.

--- a/WordPressKit.podspec
+++ b/WordPressKit.podspec
@@ -23,6 +23,6 @@ Pod::Spec.new do |s|
   s.dependency 'CocoaLumberjack', '3.4.2'
   s.dependency 'WordPressShared', '~> 1.4'
   s.dependency 'NSObject-SafeExpectations', '0.0.3'
-  s.dependency 'wpxmlrpc', '0.8.3'
+  s.dependency 'wpxmlrpc', '0.8.4'
   s.dependency 'UIDeviceIdentifier', '~> 0.4'
 end

--- a/WordPressKit.podspec
+++ b/WordPressKit.podspec
@@ -8,7 +8,7 @@ Pod::Spec.new do |s|
                     with WordPress.com and WordPress.org endpoints.
                     DESC
 
-  s.homepage      = "http://apps.wordpress.com"
+  s.homepage      = "https://github.com/wordpress-mobile/WordPressKit-iOS"
   s.license       = "GPLv2"
   s.author        = { "Jorge Leandro Perez" => "jorge.perez@automattic.com" }
   s.platform      = :ios, "10.0"

